### PR TITLE
Dimberman add ingress annotations

### PR DESCRIPTION
**Pull request by andriisoldatenko**
- fixed few typos https://github.com/astronomer/houston-api/pull/258/commits/32f0b133b5748286c4d8f7c7ff6f873a2c68223e
- practically tested in KinD

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
related PR https://github.com/astronomer/airflow-chart/pull/70

**Comment by andriisoldatenko** [comment]
@schnie PTAL ^
@sjmiller609

File: src/lib/utilities/index.js [hunk 1]
```diff
@@ -54,6 +54,47 @@ export function ui() {
...
+  return `${subdomain}.${baseDomain}`;
+}
+
+/*
+ * Return flower subdomain
+ * @return {String} The flower subdomain.
+ */
+export function flowerSubdomain() {
+  const baseDomain = config.get("helm.baseDomain");
+  const releaseName = config.get("helm.releaseName");
```
Comment (schnie) [inline src/lib/utilities/index.js:73]:
```suggestion
  const { baseDomain, releaseName } = config.get("helm");
```

File: src/lib/utilities/index.js [hunk 2]
```diff
@@ -54,6 +54,47 @@ export function ui() {
...
+  return `${releaseName}-flower.${baseDomain}`;
+}
+
+/*
+ * Return airflow subdomain
+ * @return {String} The airflow subdomain.
+ */
+export function airflowSubdomain() {
+  const baseDomain = config.get("helm.baseDomain");
+  const releaseName = config.get("helm.releaseName");
```
Comment (schnie) [inline src/lib/utilities/index.js:83]:
```suggestion
  const { baseDomain, releaseName } = config.get("helm")
```

File: src/lib/deployments/config/index.js [hunk 3]
```diff
@@ -70,10 +78,51 @@ export function generateHelmValues(deployment, values = {}) {
...
+        commonAnnotations
+      ),
+      flowerAnnotations: merge(
+        {
+          "nginx.ingress.kubernetes.io/rewrite-target": "/$2;",
+          "nginx.ingress.kubernetes.io/configuration-snippet": `
+             if ($host = '${flowerSubdomain()}' ) {
+               rewrite ^ ${deploymentsUrl()}/flower permanent;
+             }
+             subs_filter_types text/css text/xml text/css;
```
Comment (schnie) [inline src/lib/deployments/config/index.js:119]: @andriisoldatenko is this a typo? Should it be `sub_filter_types`?

**Comment by andriisoldatenko** [comment]
image
@schnie Practically LGTM! except few your suggestions

File: src/lib/deployments/config/index.js (hunk 3 above)
Comment (andriisoldatenko) [reply src/lib/deployments/config/index.js:119]: @schnie https://www.nginx.com/resources/wiki/modules/substitutions/#example
it's correct one.

**Comment by andriisoldatenko** [comment]
@schnie PTAL.

File: src/lib/utilities/index.js (hunk 1 above)
Comment (andriisoldatenko) [reply src/lib/utilities/index.js:73]: fixed.

**Pull request merged**